src/main/resources/liquibase/crop_changelog/15_3_0.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet author="aldrin" id="v15.3.0-1">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="germplasm_study_source"/>
+			</not>
+		</preConditions>
+		<comment>
+			Create new germplasm_study_source
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			CREATE TABLE `germplasm_study_source` (
+			`source_id` INT NOT NULL AUTO_INCREMENT,
+			`gid` INT NOT NULL,
+			`project_id` INT NOT NULL,
+			`nd_experiment_id` INT NULL,
+			`type` VARCHAR(45) NULL,
+			PRIMARY KEY (`source_id`),
+			INDEX `asdasd_idx` (`project_id` ASC),
+			INDEX `FK_nd_experiment_id_idx` (`nd_experiment_id` ASC),
+			CONSTRAINT `FK_project_id_1` FOREIGN KEY (`project_id`) REFERENCES `project` (`project_id`) ON DELETE NO ACTION ON UPDATE NO ACTION,
+			CONSTRAINT `FK_nd_experiment_id_1` FOREIGN KEY (`nd_experiment_id`) REFERENCES `nd_experiment` (`nd_experiment_id`) ON DELETE NO ACTION ON UPDATE NO ACTION);
+		</sql>
+	</changeSet>
+
+	<changeSet author="aldrin" id="v15.3.0-2">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="ims_stock_transaction" columnName="source_id"/>
+			</not>
+		</preConditions>
+		<comment>
+			Add source_id column
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			ALTER TABLE `ims_stock_transaction`
```

Comment:
I guess we still need to do some extra work on this area and migrate the table content to the nd_experiment_transaction table, but we still need to have some extra discussion on that regard

Reply:
I can see that the StockTransaction entity was not updated after this addition

Reply:
Done